software/pkg-lists-wmla120.yml

```diff
@@ -545,6 +545,12 @@ yum_pkgs_p8:
     - xorg-x11-xkb-utils-7.7-14.el7.ppc64le
     - yum-utils-1.1.31-50.el7.noarch
     - zlib-devel-1.2.7-18.el7.ppc64le
+# Do not remove the additional packages below. They are included
+# in the pup kickstart and are thus not detected in package dependency
+# resolution
+    - wget
```

Comment:
Can you please add these  packages to: pkg-lists-wmla120_x86_64.yml ?

Reply:
done

Reply:
verified on p8 node

Reply:
verified on x86